frontend/projects/setup-wizard/src/app/pages/transfer/transfer.page.ts

```diff
@@ -0,0 +1,74 @@
+import { Component } from '@angular/core'
+import { AlertController, NavController } from '@ionic/angular'
+import { ApiService, DiskInfo } from 'src/app/services/api/api.service'
+import { ErrorToastService } from '@start9labs/shared'
+import { StateService } from 'src/app/services/state.service'
+
+@Component({
+  selector: 'app-transfer',
+  templateUrl: 'transfer.page.html',
+  styleUrls: ['transfer.page.scss'],
+})
+export class TransferPage {
+  loading = true
+  drives: DiskInfo[] = []
+
+  constructor(
+    private readonly apiService: ApiService,
+    private readonly navCtrl: NavController,
+    private readonly alertCtrl: AlertController,
+    private readonly errToastService: ErrorToastService,
+    private readonly stateService: StateService,
+  ) {}
+
+  async ngOnInit() {
+    await this.getDrives()
+  }
+
+  async refresh() {
+    this.loading = true
+    await this.getDrives()
+  }
+
+  async getDrives() {
+    try {
+      const disks = await this.apiService.getDrives()
+      this.drives = disks.filter(d => d.partitions.length && d.guid)
+    } catch (e: any) {
+      this.errToastService.present(e)
+    } finally {
+      this.loading = false
+    }
+  }
+
+  async select(target: DiskInfo) {
+    const { logicalname, guid } = target
+
+    if (!logicalname) return
+
+    const alert = await this.alertCtrl.create({
+      header: 'Warning',
+      message:
+        'Data from this drive will not be deleted, but you will not be able to use this drive to run embassyOS after the data is transferred.',
```

Comment:
@MattDHill review copy ^

Reply:
Might want to explain the consequences of trying to use the old drive after transfer. Let's talk about that.

Reply:
We can. Aiden and I discussed a way we could mark the drive data as unusable. It would still technically be there if needed for a manual recovery, but it would be rendered useless by our normal setup flows. I forget the exact implementation he described, but it was possible.

Reply:
We should discuss, but see updated copy for now.